common/src/main/java/org/apache/nemo/common/DirectByteArrayOutputStream.java

```diff
@@ -40,18 +40,4 @@ public DirectByteArrayOutputStream() {
   public DirectByteArrayOutputStream(final int size) {
     super(size);
   }
-
```

Comment:
If we do not need these methods, why don't we just remove this class?

Reply:
I just realized we still need getbufDirectly(). I will open a new PR, thank you!